resources/precompiled-js/application.js

```diff
@@ -196,14 +192,28 @@ $(document).ready(function () {
       //show the fa questions when checked
       $("#fa-questions").fadeIn();
       $("#fa-questions :input").prop("required", true);
-      $("#how_complete_course").prop("required", false); //hide standard questions
-
-      $("#how_to_enroll").prop("checked", false);
-      $("#standard-questions").fadeOut();
-      $("#standard-questions :input").prop("required", false);
+      $("#how_complete_course").prop("required", false);
     } else {
       $("#fa-questions").fadeOut();
       $("#fa-questions :input").prop("required", false);
     }
+  });
+  $('input:radio[id="belongs_to_flood_area"]').change(function () {
+    if ($(this).is(":checked") && $(this).val() == "1") {
+      $("#flood_area_questions").fadeIn();
+      $("#flood_area_questions :input").prop("required", true);
+    } else {
+      $("#flood_area_questions").fadeOut();
+      $("#flood_area_questions :input").prop("required", false);
+    }
+  });
+  $('input:radio[id="belongs_to_flood_area_no"]').change(function () {
+    if ($(this).is(":checked") && $(this).val() == "0") {
+      $("#flood_area_questions").fadeOut();
+      $("#flood_area_questions :input").prop("required", false);
+    } else {
+      $("#flood_area_questions").fadeIn();
+      $("#flood_area_questions :input").prop("required", true);
+    }
```

Comment:
Even if we remove both of these `else` statements, the feature still works fine.